pep/app/pods/read/document/route.ts

```diff
@@ -41,9 +43,43 @@ export default class ReadDocument extends PageNav(Route) {
      * Fetch the requested document
      * @param {ReadDocumentParams} params
      */
-    model(params: ReadDocumentParams) {
+    model(params: ReadDocumentParams, transition: Transition) {
+        let searchQueryString;
+        if (params.q) {
```

Comment:
why does the search query string only get added if the `q` param (the smart text field) has a value? As i believe its valid for a user to do a search that has that blank, but has specific search term fields (the add-able fields below it in the form) populated.